### PR TITLE
adding a connection string in new DbContext scenarios

### PR DESCRIPTION
**Pull request by deepchoudhery**
added `AddConnectionString` to `DbContextHelper` and calling it in `CreateDbContext`
- find `appsettings.json` file
- parse it to a `JObject`
- find/create a `ConnectionStrings` node
- add the new connection string for the newly created DbContext

other minor changes :
- `DatabaseCommand` now actually configuring for the correct `--type` parameter.
- initialize `DbContextProperties` to include new property `NewDbConnectionString`
- remove unnecessary `CommandHelpers.AddDbContext` method and switch `MinimalApiCommand` use of it to `DbContextInfo.AddDbContext`
- removed unnecessary properties from `PackageConstants.cs`
-

### CODE REVIEW

File: src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/AspNetDbContextHelper.cs [hunk 1]
```diff
@@ -14,21 +14,25 @@ internal class AspNetDbContextHelper
...
     internal static DbContextProperties SqliteDefaults = new()
     {
         AddDbMethod = "AddSqlite",
+        NewDbConnectionString = "Data Source={0}.db"
     };
 
     internal static DbContextProperties CosmosDefaults = new()
     {
         AddDbMethod = "AddCosmos",
+        NewDbConnectionString = "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="
```
Comment (tlmii) [inline src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/AspNetDbContextHelper.cs:29]: What is this AccountKey (even if it is local only or just a placeholder I'm a little worried about it getting flagged by credscan)?

Comment (deepchoudhery) [reply src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/AspNetDbContextHelper.cs:29]: I got this localhost connection string from the official Azure Cosmos DB [docs](https://learn.microsoft.com/en-us/azure/cosmos-db/emulator) regarding running the local emulator.

File: src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Helpers/General/DbContextHelper.cs [hunk 2]
```diff
@@ -40,12 +41,62 @@ public static bool CreateDbContext(DbContextProperties dbContextProperties, stri
 
         var t4TemplateName = Path.GetFileNameWithoutExtension(t4TemplatePath);
         var templatedString = templateInvoker.InvokeTemplate(textTransformation, dictParams);
-        if (!string.IsNullOrEmpty(templatedString) && !string.IsNullOrEmpty(dbContextPath))
+        if (!string.IsNullOrEmpty(templatedString) &&
```
[the remark this answers is not in the document]
Comment (deepchoudhery) [reply src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Helpers/General/DbContextHelper.cs:44]: yep sounds good

File: src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/AspNetDbContextHelper.cs (hunk 1 above)
Comment (tlmii) [reply src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/AspNetDbContextHelper.cs:29]: Got it. Well then lets hope credscan already has it allow listed :)

**Review by tlmii — APPROVED** [review]

**Pull request merged**